index.html

```diff
@@ -19,6 +19,7 @@
                 <li><a href="#">About</a></li>
                 <li><a href="#">Contact</a></li>
             </ul>
+            <img class="hamburger" src="images/hamburger.png" alt="">
```

Comment:
- [ ] Kindly, wrap your hamburger menu with a nav tag. Means instead of using `div` tag for your navigation menu, you can use `nav` tag. do it by wrapping your menu from line 14 up 24 with a `<nav>` tag. have a look here:
```HTML
<nav class="nav">
   .... your code about the list of menu ...
   <img class="hamburger" src="images/hamburger.png" alt="">
   ...
</nav>
```

Reply:
Wrapped it. 💯